solr/prometheus-exporter/src/test/org/apache/solr/prometheus/utils/Helpers.java

```diff
@@ -46,12 +51,35 @@ public static void indexAllDocs(SolrClient client) throws IOException, SolrServe
                 ContentStreamUpdateRequest req = new ContentStreamUpdateRequest("/update");
                 try {
                   req.addFile(xml, "application/xml");
-                  client.request(req, PrometheusExporterTestBase.COLLECTION);
+                  NamedList<Object> response =
```

Comment:
@dsmiley I am a bit worried about this change.   I was getting ObjectTracker errors until I added this.  Seems a bit worrisome that we might have unclosed streams unless we actually consume them..   Is this a potential issue with our InputStreamResponseParser?   Do we need it to internally close the stream in the `STREAM_KEY` if it isn't actually read?